src/main/io/serial_4way.c

```diff
@@ -422,6 +422,11 @@ void esc4wayProcess(serialPort_t *mspPort)
     uint8_t *InBuff;
     ioMem_t ioMem;
 
+    if (!mspPort)
+    {
```

Comment:
Yes please. You can do a `git commit -a --amend`, followed by a `git push --force-with-lease`, and the fix will appear in your original commit and the PR. (Also, I did not mean you had to submit a new PR with my comment about squashing. Doing a `git rebase -i master` (selecting `pick` for the first and `squash` for all other commits) and a `git push --force-with-lease` will squash your commits into one in place.)